docs/kubernetes/how-to-deploy-kubeflow-on-linode-kubernetes-engine/index.md

```diff
@@ -0,0 +1,252 @@
+
+2.  The `kfctl` CLI used to deploy Kubeflow currently only works on Linux and Mac. If you're on Windows, you can use WSL2 or a Docker container to work around this limitation.
+
+3.  Most Kubeflow pipelines require [Kubernetes Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) that can be attached to several nodes at once (ReadWriteMany). Currently, the only mode supported by the Linode Block Storage CSI driver is ReadWriteOnce, meaning that it can only be connected to one Kubernetes node at a time.
+
+## Create an LKE Cluster
+
+Follow the instructions in [Deploying and Managing a Cluster with Linode Kubernetes Engine Tutorial](https://www.linode.com/docs/kubernetes/deploy-and-manage-a-cluster-with-linode-kubernetes-engine-a-tutorial/) to create and connect to an LKE cluster.
+
+The [official Kubeflow documentation](https://www.kubeflow.org/docs/started/k8s/overview/) recommends provisioning a cluster with at least 4 CPU, 12GB of memory and 50GB of space available.
+
+We recommend running three "Linode 16GB" nodes — that should give you enough resources to scale your models.
+
+![Creating a Kubernetes cluster on Linode](create-cluster.png)
+
+You can verify that the installation is successful with:
+
+```bash
+kubectl get nodes
+```
+
+The output should similar to:
+
+{{< output >}}
+NAME                        STATUS   ROLES    AGE   VERSION
+lke7189-9006-5f05145fc9a3   Ready    <none>   8h    v1.17.3
+lke7189-9006-5f051460a1e2   Ready    <none>   8h    v1.17.3
+lke7189-9006-5f0514617a87   Ready    <none>   8h    v1.17.3
+{{</ output>}}
+
+## Install Kubeflow
+
+To install Kubeflow, you need three parts:
+
+1. A Kubernetes cluster, which you already provisioned in the previous step.
+1. `kfctl` — a command-line tool that configures and deploy Kubeflow.
+1. A `KfDef` file — a recipe of components that should be included in your Kubeflow installation.
+
+You can [download and install `kfctl` from the official repository](https://github.com/kubeflow/kfctl/releases/).
+
+> Please notice that there is no Windows release available at the moment.
+
+You can verify that the binary is installed correctly with:
+
+```bash
+kfctl version
+```
+
+The last piece of the puzzle is the `KfDef` file.
+
+You can think of the `KfDef` file as a list of components that should be installed with Kubeflow.
+
+As an example, Kubeflow can be configured to use the [Spark operator](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator), [Katib (Hyperparameter Tuning)](https://www.kubeflow.org/docs/components/hyperparameter-tuning/hyperparameter/), [Seldon serving](https://www.kubeflow.org/docs/components/serving/seldon/), etc.
+
+You can download the following `KfDef` that includes a lighter version of Kubeflow as well as [Dex — an identity provider](https://github.com/dexidp/dex) useful for managing user access.
+
```

Comment:
Note: If you want to use a complete version of Kubeflow download the `KfDef` file from https://raw.githubusercontent.com/kubeflow/manifests/v1.0-branch/kfdef/kfctl_istio_dex.v1.0.2.yaml

Reply:
This will not work, unfortunately. We had to patch Istio and use a previous version.

Reply:
Yes, that is right does https://github.com/learnk8s/kubeflow-pipelines-demo/blob/master/kfctl_istio_dex.v1.0.2-full.yaml work?

Reply:
Ideally, I'd like to track down the issue and notify the LKE team. The issue isn't just kubeflow, but any installation of Istio.
The second best option is to use https://github.com/learnk8s/kubeflow-pipelines-demo/blob/master/kfctl_istio_dex.v1.0.2-full.yaml

Also, I can write the instructions on how to patch it if you prefer.

Spartakus reports usage to the mothership and it's installed (and enabled) by default, unfortunately.

Reply:
Yes, Please notify the LKE team about the issue. 
I like what you suggested.

Reply:
I don't have time to track down the issue at the moment. I don't understand this page enough to make a suggestion https://istio.io/latest/blog/2019/trustworthy-jwt-sds/